usr/src/uts/armv8/sys/smp_impldefs.h

```diff
@@ -87,9 +71,6 @@ extern caddr_t psm_map_phys_new(paddr_t, size_t, int);
 
 /* unmap the physical address given in psm_map_phys() from the addr	*/
 extern void psm_unmap_phys(caddr_t, size_t);
```

Comment:
I'm keeping these around for now - I'll be needing (and implementing) them soon.